assets/js/modules/adsense/dashboard/dashboard-widget-main-summary.js

```diff
@@ -109,6 +109,8 @@ class AdSenseDashboardMainSummary extends Component {
 			{}
 		);
 
+		const currency = period.headers.find( header => null !== header.currency && 0 < header.currency.length ).currency;
```

Comment:
This is subject to the same issue as #114, we should check whether the `find` function actually found something before accessing a property on the result (which may be undefined). Let's name this variable `currencyHeader` and remove the `.currency` at the end. Then, right below, do `const currencyCode = currencyHeader ? currencyHeader.currency : false;`.